instat/dlgDescribeTwoVarGraph.Designer.vb

```diff
@@ -34,9 +34,9 @@ Partial Class dlgDescribeTwoVarGraph
         '
         'cmdOptions
         '
-        Me.cmdOptions.Location = New System.Drawing.Point(10, 197)
+        Me.cmdOptions.Location = New System.Drawing.Point(311, 228)
         Me.cmdOptions.Name = "cmdOptions"

```

Comment:
I changed this to position 10,97 intentionally, based on the the discussion that led to the frequencies dialogues. Where the options buttons were moved to the left just as i did here. For consistency with most of the other dialogues as well.